src/pybind/CMakeLists.txt

```diff
@@ -68,27 +63,20 @@ if(NMODL_ENABLE_PYTHON_BINDINGS)
   if(LINK_AGAINST_PYTHON)
     target_link_libraries(_nmodl PRIVATE pywrapper)
   endif()
-
-  add_custom_command(
-    OUTPUT ${PROJECT_BINARY_DIR}/lib/nmodl
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_FILE:_nmodl> ${PROJECT_BINARY_DIR}/nmodl
-    DEPENDS ${NMODL_PYTHON_FILES_IN} _nmodl
-    COMMENT "-- COPYING NMODL PYTHON MODULE --")
```

Comment:
AFAICT the file `$<TARGET_FILE:_nmodl>` (a `_nmodl*.so`) automatically ends up at that location. This also happens when using this branch in neuron.

Moreover, it not obvious why there's a target that has a dependency on required file/folder to trigger this "custom command" to be run.

Reply:
> Moreover, it not obvious why there's a target that has a dependency on required file/folder to trigger this "custom command" to be run.

I didn't check the details but it seems to be introduced in b2c7443dc25dcfb69304ab1561dec534a1c1d4c6 with a very relevant (?) comment: 

```
         - Fixed issue in build system that prevented pytest from working
           correctly. This was a regression introduced by the recent build-
           system changes.
```

@tristan0x - you might have a better memory about this?

> AFAICT the file $<TARGET_FILE:_nmodl> (a _nmodl*.so) automatically ends up at that location

I see the same.

Reply:
I've checked:
```
pip install -r requirements.txt
pip install scikit-build

pip install -e .
pytest
```
which runs without issues.

Reply:
Ok. My guess was that the shared library copying was triggered if any of the inputs are changed. But as you pointed out, if the library is created at the same location, not sure what was need.

Anyway, I will merge this. If we see some fishy behaviour, we know where to look at :)   (@alkino , @iomaganaris)